timed/tracking/migrations/0003_auto_20170814_1509.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-14 13:09
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def migrate_start_datetime(apps, schema_editor):
+    Activity = apps.get_model('tracking', 'Activity')
+
+    for a in Activity.objects.all():
+        a.date = a.start_datetime.date()
+        a.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tracking', '0002_auto_20170731_1047'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='activity',
+            name='date',
+            field=models.DateField(auto_now_add=True, default=django.utils.timezone.now),
+            preserve_default=False,
+        ),
+        migrations.RunSQL('SET CONSTRAINTS ALL IMMEDIATE', reverse_sql=migrations.RunSQL.noop),
+        migrations.RunPython(migrate_start_datetime),
+        migrations.RunSQL(migrations.RunSQL.noop, reverse_sql='SET CONSTRAINTS ALL IMMEDIATE'),
+        migrations.RemoveIndex(
+            model_name='activity',
+            name='tracking_ac_start_d_c43640_idx',
+        ),
+        migrations.RemoveField(
+            model_name='activity',
+            name='start_datetime',
+        ),
+        migrations.AddIndex(
+            model_name='activity',
+            index=models.Index(fields=['date'], name='tracking_ac_date_bf4ad0_idx'),
+        ),
+    ]
```

Comment:
A bit ugly to do this.... we are still not productive yet so let's start the migration from scratch again. We can soon enough create such migrations scripts... :).